test/fastimage_test.exs

```diff
@@ -11,16 +15,25 @@ defmodule FastimageTest do
   @png_url "https://raw.githubusercontent.com/stephenmoloney/fastimage/master/priv/test.png"
   @gif_url "https://raw.githubusercontent.com/stephenmoloney/fastimage/master/priv/test.gif"
   @bmp_url "https://raw.githubusercontent.com/stephenmoloney/fastimage/master/priv/test.bmp"
+  @webp_vp8l_url "https://raw.githubusercontent.com/sdsykes/fastimage/master/test/fixtures/webp_vp8l.webp"
+  @webp_vp8_url "https://raw.githubusercontent.com/sdsykes/fastimage/master/test/fixtures/webp_vp8.webp"
+  @webp_vp8x_url "https://raw.githubusercontent.com/sdsykes/fastimage/master/test/fixtures/webp_vp8x.webp"
```

Comment:
we can point these urls at this repo, but left them as is so you could see the tests pass first